src/templates/common/encoding/article_jats_1_2.xml

```diff
@@ -81,6 +122,18 @@
                 {% endfor %}
             </kwd-group>
             {% endif %}
+
+            {% if article.funders.all %}
+                {% for funder in article.funders.all %}
+                    <funding-group>
+                        <award-group>
+                            <funding-source id="{{ funder.fundref_id }}">{{ funder.name }}
```

Comment:
`funder.fundref_id` can be null, which would violate jats constrains over `@id` attribute